components/selection/selection-select-all.js

```diff
@@ -51,6 +51,7 @@ class SelectAll extends LocalizeCoreElement(SelectionObserverMixin(LitElement))
 			composed: true,
 			detail: { checked: e.target.checked }
 		}));
+		if (this.subscribedTo && this._provider) this._provider.setSelectionForAll(e.target.checked);
```

Comment:
My thinking here is that we always want to send the `d2l-selection-select-all-change` event since we advertise that for consumers, but if we're external to the collection we'll need to trigger the selection ourselves.

Reply:
For simplicity then would it be worth changing the provider so that it doesn't listen for the event but instead just expects this method to be called?

Reply:
Yeah, if we need to do this anyways that would simplify things.  If no one is using the event yet, do we even still want it?  I'm working under the assumption this is an event we expect consumers to want to listen for and then do something with